source/api/discovery/0.9/index.md

```diff
@@ -896,13 +892,35 @@ Given an array (`collections`) of collection URIs as input,
 
 ## 4. Network Considerations
 
-### 4.1. Activities for Access-Restricted Content
+### 4.1. Media Type
+
+The base format for all responses of this API is JSON, as described above. 
+
+If the server receives a request with an `Accept` header, it _SHOULD_ respond following the rules of [content negotiation][org-rfc-7231-conneg]. Note that content types provided in the `Accept` header of the request _MAY_ include parameters, for example `profile` or `charset`.
+
+If the request does not include an `Accept` header, the HTTP `Content-Type` header of the response _SHOULD_ have the value `application/ld+json` (JSON-LD) with the `profile` parameter given as the context document: `http://iiif.io/api/discovery/1/context.json`.
+
+``` none
+Content-Type: application/ld+json;profile="http://iiif.io/api/discovery/1/context.json"
+```
+{: .urltemplate}
+
+If the `Content-Type` header `application/ld+json` cannot be generated, then the `Content-Type` header _SHOULD_ instead be `application/json` (regular JSON), without a `profile` parameter.
```

Comment:
In the interest of clarity that we prefer `application/json` over `application/ld+json` without `profile`, should this instead say:

> If the `Content-Type` header `application/ld+json` cannot be generated **with the `profile` parameter**, then the `Content-Type` header _SHOULD_ instead be `application/json` (regular JSON), without a `profile` parameter.